src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -83,7 +83,9 @@ public function build(ContainerBuilder $container)
         if (class_exists(AddConsoleCommandPass::class)) {
             $container->addCompilerPass(new AddConsoleCommandPass());
         }
-        $container->addCompilerPass(new FormPass());
+        if (class_exists(FormPass::class)) {
```

Comment:
As this going to be very big would it make sense to use a private helper method?
`$this->addExistingCompilerPass($className, $arguments = array())`.

So you don't end-up with numerous if-statements 👍